src/test/java/nl/tudelft/scrumbledore/ConstantsTest.java

```diff
@@ -0,0 +1,36 @@
+package nl.tudelft.scrumbledore;
+
+//import static org.junit.Assert.*;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+//import org.junit.Test;
+
+/**
+ * Testing the Constants class.
+ * 
+ * @author Floris Doolaard
+ *
+ */
+public class ConstantsTest {
```

Comment:
Testing the Constants class seems to be unnecessary to me as it only contains static final variables and not methods or changing values.